config/dev/settings.example.json

```diff
@@ -0,0 +1,4 @@
+{
+  "url": "https://example.com/path/to/the/config/endpoint",
+  "api_key": "123"
```

Comment:
API key should not be in settings, and needs to be an ENV Var.
Otherwise its not secure when used in production.